lib/src/pub_updater.dart

```diff
@@ -69,12 +72,18 @@ class PubUpdater {
         'activate',
         packageName,
         if (versionConstraint != null) versionConstraint,
+        if (_baseUrl != _defaultBaseUrl) ...[
+          '--hosted-url',
+          _baseUrl,
+          '--source',
+          'hosted'
+        ]
```

Comment:
Just a nit, but technically we dont need this if statement right? It will work either way

Reply:
technically you're right, we don't need this if statement. but, do we need to override the default value with the same value? but as you wrote, it'll work either way. i can create yet another PR and remove this if statement :)